build-tools/packages/build-cli/src/commands/release.ts

```diff
@@ -0,0 +1,834 @@
+                            cmd: `${this.config.bin} ${this.id} -g ${this.releaseGroup}`,
+                        });
+                    }
+                }
+
+                await this.writePrompt(prompt);
+                this.exit();
+                break;
+            }
+
+            case "CheckShouldCommitBump":
+            case "CheckShouldCommitDeps": {
+                if (!this.shouldCommit) {
+                    this.machine.action("failure");
+                    break;
+                }
+
+                const version = this.releaseVersion;
+                const newVersion = bumpVersionScheme(version, this.bumpType!);
+
+                const branchName = generateBumpVersionBranchName(
+                    this.releaseGroup!,
+                    this.bumpType!,
+                    this.releaseVersion!,
+                );
+
+                await context.createBranch(branchName);
+
+                this.verbose(`Created bump branch: ${branchName}`);
+
+                const commitMsg = `[bump] ${this.releaseGroup}: ${version} => ${newVersion} (${this.bumpType})\n\nPost-release ${this.bumpType} bump of ${this.releaseGroup}.`;
+                await context.gitRepo.commit(commitMsg, `Error committing to ${branchName}`);
+                this.machine.action("success");
+                break;
+            }
+
+            case "CheckShouldCommitReleasedDepsBump": {
+                if (!this.shouldCommit) {
+                    this.machine.action("success");
+                }
+
+                assert(isReleaseGroup(this.releaseGroup));
+                const branchName = generateBumpDepsBranchName(this.releaseGroup, "releasedDeps");
+                await context.gitRepo.createBranch(branchName);
+
+                this.verbose(`Created bump branch: ${branchName}`);
+                this.log(
+                    `BUMP: ${this.releaseGroup}: Bumped prerelease dependencies to release versions.`,
+                );
+
+                const commitMsg = `[bump] ${this.releaseGroup}: update prerelease dependencies to release versions`;
+                await context.gitRepo.commit(commitMsg, `Error committing to ${branchName}`);
+                this.machine.action("success");
+                break;
+            }
+
+            case "PromptToCommitBump": {
+                const prompt: InstructionalPrompt = {
+                    title: "NEED TO COMMIT LOCAL CHANGES",
+                    sections: [
+                        {
+                            title: "FIRST",
+                            message: `Commit the local changes and create a PR targeting the ${context.originalBranchName} branch.\n\nAfter the PR is merged, then the release of ${this.releaseGroup} is complete!`,
```

Comment:
Nit: looks like this string is duplicated below. Can we share a getter property or something?